build/scons.js

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/opt/homebrew/bin/node
```

Comment:
I think this should stay unchanged?
```suggestion
#!/usr/bin/env node
```